v5_ws_public.go

```diff
@@ -71,6 +79,9 @@ const (
 
 	// V5WebsocketPublicTopicTicker :
 	V5WebsocketPublicTopicTicker = V5WebsocketPublicTopic("tickers")
+
+	// V5WebsocketPublicTopicTrade :
+	V5WebsocketPublicTopicTrade = V5WebsocketPublicTopic("publicTrade")
```

Comment:
nits:

To unify with the others

```suggestion
	V5WebsocketPublicTopicTrade = V5WebsocketPublicTopic("trade")
```

Reply:
This was as it should be.

<img width="464" alt="スクリーンショット 2023-10-05 8 58 17" src="https://github.com/hirokisan/bybit/assets/12891407/41d7292a-ba53-4317-ac6d-9cbf4dae98f9">